### ISSUE TITLE
[1.7.0] DAO gets out of sync once or twice a day

### ISSUE THREAD

**Issue by Akira45-0** [bisq-network/bisq#5595]
### Description

Since the update to 1.7.0 I needed to make DAO resync four times, which means that the DAO needs to be resynced once or twice a day.

I am using Bisq on Windows 10 as a full DAO node.

#### Version

1.7.0

### Steps to reproduce

Just use Bisq. Sooner or later it will ask you to shut down and resync the DAO.

### Expected behaviour

DAO should remain synced.

### Actual behaviour

DAO gets out of sync.

#### Device or machine

Windows 10.

#### Additional info

**Comment by bisqubutor** [comment]
![Your local data is not in consensus with at least one seed node. Please resync the DAO state.](https://user-images.githubusercontent.com/79169291/124075985-2c857e00-da35-11eb-9efd-5c58e0c33a15.png)


`Your local data is not in consensus with at least one seed node. Please resync the DAO state.`

**Comment by tsjk** [comment]
Yes, I've stumbled upon this as well. Twice within 6 hours or so.
My operating system is Linux.

**Comment by ripcurlx** [comment]
I got it myself a couple of times already as well. Looking into it.

**Comment by PicaroonEsp** [comment]
Me too, in 10 hours 2 times

**Comment by jkuchar** [comment]
Also affected. Directly after resync I get message that DAO is out of sync. It looks like there is 100% conflicts with other nodes. It looks like my node is missing any peers consensus after block 689 196. Conflicts starts after block 690 300.

When I open "Network monitor" I get message:

> Your local data is not in consensus with at least one seed node. Please resync the DAO state.

**Comment by ddawson** [comment]
Having the same thing happening in 1.8.0, on Linux. Not a full node. If it helps, I'm running my own Bitcoin node and an external (to Bisq) Tor daemon.

**Comment by ripcurlx** [comment]
> 

We do have a seednode that is out of sync ATM which is filtered out, but you need to restart your client once to be able to apply this filter also on startup.

**Comment by ddawson** [comment]
Do you mean the filter is applied automatically? Because I've restarted the client quite a few times, and this keeps happening.

**Comment by ripcurlx** [comment]
> Do you mean the filter is applied automatically? Because I've restarted the client quite a few times, and this keeps happening.

🤔  Can you please check the DAO network monitor and let me know with which seednodes you have a conflict with.

**Comment by ddawson** [comment]
> Can you please check the DAO network monitor and let me know with which seednodes you have a conflict with.

None at the moment:

> One of your peers is not in consensus with the network but your node is in sync with the seed nodes.

That is part of what I have been observing. I'll keep an eye on it to see if the other part (saying it needs to be resynced) happens again.

**Comment by ddawson** [comment]
It happened again. It won't tell me which node(s), or at least I don't know how to get that info. (Could it be because "Full-mode DAO state monitoring" is off?) It just says, "Your local data is not in consensus with at least one seed node. Please resync the DAO state."

It may be worth noting, however, under "Chain of DAO state hashes", even when it doesn't show a conflict with seed nodes, there are many blocks showing conflicts. At this point, the last 283 blocks (718061–718343) have anywhere from 2 to 13 conflicts each. Hash creator is "self" for all of them, while for those without conflicts (except the last), it's "peer". It appears as though it's building up conflicts with peers, until eventually it conflicts with a seed.

**Comment by golyalpha** [comment]
I have a fresh install of Bisq 1.8.0 on Linux (Fedora 35), this is happening consistently every time I start it (even after requesting the resync) I have conflicts with all the nodes listed.
![image](https://user-images.githubusercontent.com/8203992/149323179-6aefa344-ee7b-4955-9b7f-773d63423f19.png)

**Comment by golyalpha** [comment]
Whatever the issue was, after updaing to v1.8.2 and doing a DAO resync this appears to no longer be an issue for me.

**Comment by snex** [comment]
v1.9.14, this has started happening to me recently. On Settings > Network Info, I see data being sent and received but it seems extremely slow. Eventually the app just gives up and says I'm out of sync.

**Comment by bitdisturber** [comment]
Ditto here.  v1.9.14, Win 11, message to resync every few hours, has been happening for weeks (coudn't say whether it would have been an issue prior to v1.9.14 as I was away for an extended period and restarted at v1.9.14).

**Comment by alecov** [comment]
Twice a day? More like, five times per ten minutes, amirite?

It is so annoying I can restart the client five times in a row and it will almost surely and immediately complain about it at the sixth time. This is completely bogus, since this DAO thing doesn't seem to affect trading at all, but the nagging _never ever ever_ stops.

**Assuming it doesn't affect trading**, the real fix would be: it should be completely optional for the client to sync the DAO state. I don't care about it, specially if the client won't stop nagging about it. In this case, regardless of any sync issue the client should _never_ popup on your face, or at the very least have a checkbox "Do not show me this ever again".

**Comment by pazza83** [comment]
Syncing the DAO is needed for the creation of the delayed payout transactions.

For this reason clients always need to be in sync with the DAO. The DAO sync has been a lot more stable recently. Is this still an issue?

**Comment by Akira45-0** [comment]
For me it works fine now, thank you.

**Comment by ahmogit** [comment]
Seeing DAO resync/restart popup several times/day recently, running 1.9.19, Linux. Painful to have to keep restarting.

This frequency of DAO resync is a relatively recent phenomenon; prior to a few weeks ago (early June, 2025 or so) would see DAO resync popup perhaps once a month or even less.

**Comment by martinbarilik** [comment]
I get this now every time my order is taken or i take any order. 

It is frustrating .... any idea why is this happening ?
